lib/firehose/long_poll_transport.coffee

```diff
@@ -108,7 +108,7 @@ class LongPollTransport extends Transport
 
     unless @_needToNotifyOfReconnect or @_stopRequestLoop
       @_needToNotifyOfReconnect = true
-      @config.disconnected()
+      @config.disconnected(jqXhr, status, error)
```

Comment:
Yeah, I was on the fence about this. I think it's a bit different across transports, since this is the result from a rejected jQuery promise whereas the WebSockets one is a `CloseEvent` specific to WebSockets.